src/prpy/base/manipulator.py

```diff
@@ -44,21 +44,29 @@ def __dir__(self):
         # Add planning methods to the tab-completion list.
         method_names = set(self.__dict__.keys())
         method_names.update(self.GetRobot().planner.get_planning_method_names())
+        method_names.update(self.GetRobot().actionlibrary.get_actions())
         return list(method_names)
 
     def __getattr__(self, name):
         # We have to manually perform a lookup in InstanceDeduplicator because
         # __methods__ bypass __getattribute__.
         self = bind.InstanceDeduplicator.get_canonical(self)
-        delegate_method = getattr(self.GetRobot().planner, name)
-
+ 
         # Resolve planner calls through the robot.planner field.
         # FIXME: We need to replicate the _PlanWrapper functionality here.
         if self.GetRobot().planner.has_planning_method(name):
```

Comment:
Don't bother replicating the `_PlanWrapper` logic for now. It sounds like we may be refactoring how that works in the near future; see #73.